gengokit/httptransport/templates_test.go

```diff
@@ -201,7 +197,7 @@ func DecodeHTTPSumZeroRequest(_ context.Context, r *http.Request) (interface{},
 `
 	if got, want := str, desired; got != want {
```

Comment:
I can understand this, but this is a case I haven't really encountered; and the provided case ignores the "especially" clause in my objection above, when the "expected"/"want" value is already contained in a variable. It almost always is in my tests, as I'm typically using [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests) and ranging over the slice of test cases.

If, for the sake of consistency, we wish to use this `got, want :=` pattern in our Truss tests, especially if they do not map nicely to being table driven, then I shan't object further. It's not terribly offensive and there's nothing actually *wrong* with it; It's just something I haven't seen and wouldn't think about doing myself.